CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   > Each element of this array must match *at least one* of the following
   > schemas
 
+- Create Nix flake
```

Comment:
Well, it's not really library version relevant. That's just development tool.

Reply:
Removed.